lms/djangoapps/student_enrollment/enrollment.py

```diff
@@ -143,6 +139,22 @@ def enroll(self):
             )
             enrollment_status.save()
 
+            # Update student profile (Contacts) in Zoho
+            today = current_date_for_crm()
```

Comment:
Not sure this needs a separate function since this is the only place it is used.

Reply:
I know, I just wanted to make it cleaner as a helper function, so as not to meddle with datetimes inside the enrolment script itself.